components/ciphertool/src/main/java/org/wso2/ciphertool/utils/KeyStoreUtil.java

```diff
@@ -35,7 +41,7 @@ public class KeyStoreUtil {
      * Initializes the Cipher
      * @return cipher cipher
      */
-    public static Cipher initializeCipher() {
+    public static Cipher initializeCipher(String providerName) {

```

Comment:
Might need to check any other use cases of this public static util method before fully removing the method. Best way is to overload the method to maintain backward compatibility.